server/src/main/java/com/genymobile/scrcpy/GameController.java

```diff
@@ -0,0 +1,423 @@
+package com.genymobile.scrcpy;
+
+import java.util.Arrays;
+import java.util.List;
+import com.sun.jna.Library;
+import com.sun.jna.Platform;
+import com.sun.jna.Native;
+import com.sun.jna.Structure;
+import com.sun.jna.Pointer;
+
+public final class GameController {
+    public static final int DEVICE_ADDED = 0;
+    public static final int DEVICE_REMOVED = 1;
+
+    private static int UINPUT_MAX_NAME_SIZE = 80;
+
+    public static class input_id extends Structure {
```

Comment:
`checkstyle` is complaining a lot about the naming scheme I used for these `Structure`s. I initially wanted to keep the naming identical to the headers, but it may be best to use the proper naming.

Should I keep these names and add annotations like
```
@SuppressWarnings("checkstyle:MemberName")
```
or just change them to match (eg. `input_id` -> `InputId`)?

Reply:
I have renamed the `Structure`s to match the Java naming conventions, otherwise the code would become a mess, full of `@SuppressWarnings`.

I had to use `@SuppressWarnings("checkstyle:VisibilityModifier")`, because the structure members need to be public.